.github/workflows/ci.yml

```diff
@@ -32,18 +32,65 @@ jobs:
         cd build
         cmake ..
         make
```

Comment:
Should we change the CMake build commands to be the same as what's in the README?

Reply:
Yeah, good idea. I'll also parametrize the build type.

Reply:
(I don't believe the `--config Release` args are necessary after the configure step, so I'll remove those from the README).